tests/specs/session-replay/preload.e2e.js

```diff
@@ -28,4 +28,47 @@ describe.withBrowsersMatching(notIE)('Session Replay Payload Validation', () =>
     testExpectedReplay({ data: harvestContents })
     expect(wasPreloaded).toEqual(true)
   })
+
+  it('should allow for preload if configured', async () => {
+    await browser.url(await browser.testHandle.assetURL('rrweb-instrumented.html', config({ session_replay: { preload: true } })))
```

Comment:
Preload determination also relies on session; where/how is the session state getting set to FULL/OFF in these tests?

Reply:
preload doesnt solely rely on session.  it can early load if configured to do so regardless of session state